react-native/.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
+    "parser": "babel-eslint",
+    "extends": ["standard", "standard-react"],
+    "rules": {
```

Comment:
we override here. standard made their own plugins to allow this to be loose allowing 0/1 but instead we fallback to the eslint version of one or the other